spec/support/holdings_tables.rb

```diff
@@ -2,18 +2,20 @@
 require "services"
 
 RSpec.shared_context "with tables for holdings" do
-  let(:db) { Services.holdings_db }
+  let(:db_rw) { Services.holdings_rw_db }
+  let(:db_ro) { Services.ht_ro_db }
```

Comment:
It should be obvious after a moment's reflection that this DB is read-only outside the test context, but viewed in isolation it's momentarily confusing that we wrap this in a transaction. Might be worth a single line comment. Or, the naming might be suspect since the point is not that the DB is `ro` but rather that it's the `ht` database.

Reply:
Agreed, this is rather confusing, at least for the `:let` statements. We may not really need to even include the `rw` vs. `ro` in `Services` as such.

Reply:
I have changed this to omit the `rw` vs. `ro` in the names here.